dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -250,6 +250,11 @@ DSRA4005.invalid.logintimeout=DSRA4005E: The server does not support the loginTi
 DSRA4005.invalid.logintimeout.explanation=The loginTimeout property is not supported when it is specified for a DataSource that is backed by a java.sql.Driver implementation because the server is unable to call getLoginTimeout on the java.sql.Driver class.
 DSRA4005.invalid.logintimeout.useraction=Modify the DataSource configuration to use a DataSource implementation instead of a java.sql.Driver implementation, or remove the loginTimeout property.
 
+# {0} name of the DataSource, {1} the vendor class name, {2} the start of the JDBC URL
+DSRA4006.null.connection=DSRA4006E: DataSource {0} is unable to connect to the database because {1} returned a null connection for the URL beginning with {2}.
+DSRA4006.null.connection.explanation=Although the java.sql.Driver indicated it accepted the supplied URL, it returned a null connection.
+DSRA4006.null.connection.useraction=This indicates a bug in the JDBC driver. Report the error to the JDBC driver vendor. Also ensure the JDBC driver and DataSource configuration are correct.
```

Comment:
Please correct me if this is wrong, but it seemed that the info about the error indicating a bug in the JDBC driver might fit better in Explanation instead of User Action. I also have some small style edit suggestions. How do these look to you?

DSRA4006.null.connection.explanation=Although the java.sql.Driver indicated **that** it accepted the supplied URL, it returned a null connection. This **error** indicates a bug in the JDBC driver.


DSRA4006.null.connection.useraction= Report the error to the JDBC driver vendor and ensure **that** the JDBC driver and DataSource configuration are correct.

Reply:
Hi @lzdanski.  Those updates look good.

Reply:
@lzdanski - I've made the requested updates.  Thanks!